src/foam/lib/parse/Eof.java

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
Use:
/**
 * @license
 * Copyright 2021 The FOAM Authors. All Rights Reserved.
 * http://www.apache.org/licenses/LICENSE-2.0
 */